docs/alerts.rst

```diff
@@ -68,6 +68,29 @@ You can also filter on different kind of **TTPs** (*Tools Techniques Procedures*
     ...
 
 
+Retrieving Alerts for Multiple Organizations
+--------------------
+
+With the example below, you can retrieve alerts for multiple organizations.
+
+Create a csv file with values that match the profile names in your credentials.cbc file. You can modify the code inside the for loop to fit your needs.
+
+.. code-block:: python
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> from cbc_sdk.platform import BaseAlert
+    >>> import csv
+    >>> file = open ("myFile.csv", "r", encoding='utf-8-sig')
+    >>> org_list = list(csv.reader(file, delimiter=","))
+    >>> file.close()
+    >>> for org in org_list:
+    ...     org = ''.join(org)
+    ...     api = CBCloudAPI(profile=org)
+    ...     alerts = api.select(BaseAlert).set_minimum_severity(7)[:5]
+    ...     print(alerts[0].id, alerts[0].device_os, alerts[0].device_name, alerts[0].category)
```

Comment:
A suggestion for line 90:
For extra convenience, print the Org that is being searched.
Don't assume there'll always be one alert.

    print('Results for Org {}'.format(org))
    for alert in alerts:
        print(alert.id, alert.device_os, alert.device_name, alert.category)